src/components/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Button = ({className, onClick, text, img, disabled}) => (
+  <button
+    type="button"
+    className={className}
+    onClick={onClick}
+    disabled={disabled}
+  >
+    {img && <img className={img.className} src={img.src} alt={img.alt}/>}
```

Comment:
Look how Buttons are implemented and used in React Toolbox and Material UI libraries.

http://react-toolbox.io/#/components/button
https://github.com/react-toolbox/react-toolbox/blob/dev/components/button/Button.js

https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Button/Button.js
https://material-ui.com/components/buttons/

Reply:
mission complete!